sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -107,6 +107,7 @@ package object dsl {
       UnresolvedExtractValue(expr, Literal(fieldName))
 
     def cast(to: DataType): Expression = Cast(expr, to)
+    def cast(to: DataType, timeZoneId: String): Expression = Cast(expr, to, Option(timeZoneId))
```

Comment:
I used it [here](https://github.com/apache/spark/pull/16308/files#diff-e8dddba2915a147970654aa93bee30a7R264), but we can use `Cast` directly.
I'll modify it.